hasura/metadata/databases/default/tables/public_app_metadata.yaml

```
@@ -75,6 +75,7 @@ insert_permissions:
         - integration_url
         - is_allowed_unlimited_notifications
         - is_developer_allow_listing
+        - is_higher_risk
```

sorry can you rename this per https://github.com/worldcoin/app-backend-main/pull/3774/files
```suggestion
        - android_only
```